Frontend/src/main/java/com/harmonia/client/ServerClient.java

```diff
@@ -80,5 +80,18 @@ public ServerPO getServerById(int id) {
         ResponseEntity<ServerPO> response = restTemplate.exchange(SERVER_LIST_BY_ID_URL, HttpMethod.GET, request, ServerPO.class, urlParameters);
         return response.getBody();
     }
+
+    /**
+     * Returns the server with the specified ID.
```

Comment:
This seems wrong

Reply:
Should it not say "Returns an array of servers with the specified category"?

Reply:
fixed, thanks for noticing